cmd/bosun/sched/sched.go

```diff
@@ -572,6 +576,7 @@ func (s *Schedule) save() {
 		}
 		tostore[name] = f.Bytes()
 		log.Printf("wrote %s: %v", name, conf.ByteSize(cw.written))
+		collect.Put("bosun.statefile.size", opentsdb.TagSet{"object": name}, cw.written)
```

Comment:
collect is already initialized with the `bosun` namespace, you can omit the `bosun.`. same goes below.